packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -107,8 +107,10 @@ export class DesktopInitializer {
           DesktopInitializer._initializing = false;
           DesktopInitializer._cancel = undefined;
         });
-    } else if (!this._initialized) {
-      this._initialized = Promise.resolve();
+    } else if (this._initializing) {
+      if(!this._initialized){
```

Comment:
ensure that you have the prettier extension installed and ensure that your files are formatted. Maybe [turn on format on save](https://linuxpip.org/vscode-format-on-save/).

Reply:
it is installed. I am looking for where the issue could be.